### DISCUSSION TITLE
Separate base url for assets

### DISCUSSION THREAD

**Discussion by reidransom** [ElMassimo/jekyll-vite#15] [Show and tell]
@ElMassimo Thank you for this gem!  HMR is so nice to work with.

I created a fork with support for serving the assets from a separate domain (ex: a cdn) here- https://github.com/reidransom/jekyll-vite

I added post install instructions but is there a way to automate updating the `vite.config.ts` file during install?

Maybe there's a better way to do this altogether :)

**Comment by ElMassimo** [comment]
Hi Reid!

I haven't used it outside of a Rails context, but taking a quick look at the code I think [`assetHost`](https://vite-ruby.netlify.app/config/index.html#assethost) might do the trick for any Ruby project as well.

Have you tried configuring it?

**Reply by reidransom** [reply]
That's not working for me but seems like it should.  I'll see if I can figure out where it's going wrong.
